backend/src/main/java/pl/edu/pw/elka/etherscan/EthereumAddressValidator.java

```diff
@@ -0,0 +1,23 @@
+package pl.edu.pw.elka.etherscan;
+
+
+
+public class EthereumAddressValidator {
+
+    public boolean checkAddres(String address){
+        if(address.charAt(0) != '0' || address.charAt(1) != 'x')
+            return false;
+        if(address.length() != 42)
+            return false;
+        String tmpAddress = address.replaceFirst("0x", "");
```

Comment:
tmp to raczej slaba nazwa, moze cos w stylu addressWithout0x - mowi troszke wiecej niz tmpAddress